### ISSUE TITLE
PylintCommand Computes Wrong rc-file Directory

### ISSUE THREAD

**Issue by tbpassin** [leo-editor/leo-editor#4160]
See PR #4163.

The `get_rc_file` method in the PylintCommand class of checkerCommands.py computes one of the fallback rc-file paths incorrectly:

```
    table = (
        # In the directory containing the outline.
        g.finalize_join(local_dir, base1),
        g.finalize_join(local_dir, base2),
        # In ~/.leo
        g.finalize_join(g.app.homeDir, '.leo', base1),
        g.finalize_join(g.app.homeDir, '.leo', base2),
        # In leo/test
        g.finalize_join(g.app.loadDir, '..', '..', 'leo', 'test', base2),  # <--- should be base1
        g.finalize_join(g.app.loadDir, '..', '..', 'leo', 'test', base2),
    )
```

**Comment by edreamleo** [comment]
@tbpassin Many thanks for this report. I apologize for the confusion it may have caused you. I'll fix this bug immediately.

**Comment by edreamleo** [comment]
Closed via PR #4163.
